docs/data/data-grid/state/RestoreStateInitialState.tsx

```diff
@@ -14,11 +14,13 @@ import {
 } from '@mui/x-data-grid-pro';
 import { useDemoData } from '@mui/x-data-grid-generator';
 
-function GridCustomToolbar({
-  syncState,
-}: {
-  syncState: (stateToSave: GridInitialState) => void;
-}) {
+declare module '@mui/x-data-grid' {
```

Comment:
If `pro` and `premium package module augmentation are mixed in a single project—it goes haywire. 🙈